src/providers/sh/commands/certs.js

```diff
@@ -21,13 +21,17 @@ const logo = require('../../../util/output/logo')
 
 const help = () => {
   console.log(`
-  ${chalk.bold(`${logo} now certs`)} <ls | create | renew | replace | rm> <cn>
+  ${chalk.bold(`${logo} now certs`)} [options] <command>
 
-  ${chalk.dim('Note:')}
+  ${chalk.yellow('NOTE:')} This command is intended for advanced use only. By default,
```

Comment:
Maybe put `By default,` combined with the line below.

Reply:
✅